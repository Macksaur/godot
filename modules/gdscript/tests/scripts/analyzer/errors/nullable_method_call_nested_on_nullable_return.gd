class A:
    var bar: String? = null
    var baz: A? = null
    var barz: A = null
    func get_baz() -> A?:
        return null
func test():
    var foo: A = null
    foo.barz.get_baz().bar