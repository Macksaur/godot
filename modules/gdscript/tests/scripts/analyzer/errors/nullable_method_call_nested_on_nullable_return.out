GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable return of "get_baz()" to access "bar".
